### DISCUSSION TITLE
Bug? Does not work on external drive for output folder

### DISCUSSION THREAD

**Discussion by TmyMue** [TheLastGimbus/GooglePhotosTakeoutHelper#282] [General]
Not sure if I missed something. 
I tried it several times and it seems to be an issue if you select an output folder which is on an extern drive (I uses the latest gpth-windoza.exe [v3.4.3]).

**Comment by TheLastGimbus** [comment]
I don't know what's your isssue since you gave 0 screenshots, log messages or anything that i could work on, but:

From https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper?tab=readme-ov-file#4-run-gpth :

> Fun fact: gpth moves files around by default - so if anything goes wrong mid-way, re-export the zips again 🙏

gpth moves files so you don't need any extra drive - it will take same amount of space at all time of running :+1:

**Comment by TheLastGimbus** [comment]
Duplicate od #251
